packages/core/src/services/datasets/preview.ts

```diff
@@ -0,0 +1,21 @@
+import { Dataset } from '../../browser'
+import { DiskWrapper } from '../../lib/disk'
+import { syncReadCsv } from '../../lib/readCsv'
+
+/**
+ * This service pick the first N rows of a CSV file
```

Comment:
csv lib we use also has `from` so we can implement some kind of pagination in the future